data-prepper-plugins/event-json-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/eventjson/EventJsonOutputCodecConfig.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.codec.eventjson;
+
+public class EventJsonOutputCodecConfig {
```

Comment:
Do we need this?

Reply:
Just keeping it for future use where we may want to control what fields should be included in the output.